.vsts-ci.yml

```diff
@@ -0,0 +1,155 @@
+resources:
+- repo: self
+  clean: true
+queue:
+  name: VSEng-MicroBuildVS2017
+  timeoutInMinutes: 360
+  demands: 
+  - msbuild
+  - visualstudio
+  - DotNetFramework
+
+variables:
+  BuildPlatform: 'Any CPU'
+steps:
+- task: NuGetCommand@2
+  inputs:
+    command: custom
+    arguments: 'locals all -clear'
+
+- task: ms-vseng.MicroBuildTasks.30666190-6959-11e5-9f96-f56098202fef.MicroBuildSigningPlugin@1
+  inputs:
+    signType: real
+  condition: and(succeeded(), in(variables['PB_SignType'], 'test', 'real'))
+
+- task: ms-vseng.MicroBuildTasks.32f78468-e895-4f47-962c-58a699361df8.MicroBuildSwixPlugin@1
+  inputs:
+    feedSource: 'https://devdiv-test.pkgs.visualstudio.com/_packaging/MicroBuildToolset/nuget/v3/index.json'
+
+- task: NuGetRestore@1
+  inputs:
+    solution: 'build\ToolsetPackages\InternalToolset.csproj'
+    feed: '8f470c7e-ac49-4afe-a6ee-cf784e438b93'
+
+- task: CmdLine@1
+  inputs:
+    filename: mkdir
+    arguments: 'Binaries\$(BuildConfiguration)'
+
+- task: VSBuild@1
+  inputs:
+    solution: 'src/Tools/MicroBuild/Build.proj'
+    vsVersion: 15.0
+    msbuildArgs: >- 
+        /p:TreatWarningsAsErrors=true 
+        /p:DeployExtension=false 
+        /p:TrackFileAccess=false
+        /p:OfficialBuild=true 
+        /p:VisualStudioVersion=14.0 
```

Comment:
After splitting out it becomes quite obvious that values like this are wrong. For now I'm keeping them but that's just to keep the diff between YAML and real build minimul until we get some insertions going. Once we have that I will clean up this and other nonsense options.

Reply:
https://github.com/dotnet/roslyn/issues/26354